ini/example_ini/example_config.ini

```diff
@@ -1,7 +1,7 @@
 [general_settings]
 sim_type = yue
 holding_time = 0.2
-arrival_rate = {'start': 50, 'stop': 100, 'step': 50}
+erlangs = {'start': 50, 'stop': 100, 'step': 50}
```

Comment:
Same here for example config, we'll need `sim_type` to be Arash if you want to include `erlangs`